### ISSUE TITLE
Duplicate Scripts in header

### ISSUE THREAD

**Issue by BurnedChris** [Mergifyio/react-crisp#33]
Hello, when I look at my header file I see multiple connections to crisp.

I am wondering if there is a duplication bug or something?

![image](https://user-images.githubusercontent.com/15834048/100234768-7815fd80-2f23-11eb-915f-8984bf37486e.png)

Kind regards Chris

**Comment by GuillaumeOj** [comment]
Hi Chris,

Thanks for your report.

Can you confirm to us that your are using the release 0.1.2?
And then, is it possible for you to give us a piece of code so that we can reproduce the bug?

**Comment by BurnedChris** [comment]
Hello,

this is the code that i am currently using. 

Also, I am on 0.1.2.

```js
{process.env.NODE_ENV !== 'development' && isAuthenticated ? (
        <Crisp
          crispWebsiteId="xxxx" // Required
          crispTokenId={currentUser?.auth?.issuer}
          attributes={{
            'user:company': [currentUser?.currentTeam?.name],
            'user:email': [currentUser?.user?.email],
            'user:nickname': [
              `${currentUser?.user?.firstName} ${currentUser?.user?.secondName}`,
            ],
            'user:avatar': [currentUser?.user?.avatarUrl],
          }}
          crispRuntimeConfig={{
            session_merge: true,
          }}
          safeMode
        />
      ) : (
        <Crisp
          crispWebsiteId="xxxx" // Required
          crispRuntimeConfig={{
            session_merge: true,
          }}
          safeMode
        />
      )}
```

**Comment by BurnedChris** [comment]
it seems to have been fixed (i don't think I pushed 0.1.2 to my server)

**Comment by GuillaumeOj** [comment]
Cool!

Have fun with it!
